### PR TITLE
Closes #5075 Remove RUCSS resources table

### LINKED ISSUE #5075: Drop RUCSS resources table completely with old RUCSS shutdown date
**Is your feature request related to a problem? Please describe.**
With old RUCSS shutdown at 30th of May 2022, we need to completely delete the resources table.

**Describe the solution you'd like**
With update we need to run raw query to drop this table and delete all related table class and schema from the code.
Don't forget the delete the table version row from options table that's added by BerlinDB.

**Describe alternatives you've considered**
Even if for any reason the user rolled back to older version, this table will be created (not to trigger any error on the customer's site) but the feature itself will not work.

**Additional context**
None

### PR DESCRIPTION
**Pull request by remyperona**
## Description

Remove the RUCSS resources table and all the related classes usage in the plugin

Fixes #5075

## Type of change

- [x] Enhancement (non-breaking change which improves an existing functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Database/deleteOldUsedCss.php [hunk 1]
```diff
@@ -25,14 +23,10 @@ public function setUp() : void {
 			return;
 		}
 
-		$this->resources = $this->getMockBuilder('WP_Rocket\Engine\Optimization\RUCSS\Database\Tables\Resources')
-			->disableOriginalConstructor()
-			->getMock();
-
 		$this->usedCSS = $this->getMockBuilder('WP_Rocket\Engine\Optimization\RUCSS\Database\Tables\UsedCSS')
```
Comment (CrochetFeve0251) [inline tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Database/deleteOldUsedCss.php]: UsedCSS::class would be cleaner here?

File: tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Database/dropResourcesTable.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	protected function setUp(): void {
+		parent::setUp();
+
+		if ( $this->isPHP8() ) {
+			return;
+		}
+
+		$GLOBALS['wpdb'] = new wpdb();
+
+		$this->usedCSS  = $this->getMockBuilder( 'WP_Rocket\Engine\Optimization\RUCSS\Database\Tables\UsedCSS' )
```
Comment (CrochetFeve0251) [inline tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Database/dropResourcesTable.php]: Same here

File: tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Database/dropRucssDatabaseTables.php [hunk 3]
```diff
@@ -25,14 +23,10 @@ public function setUp() : void {
 			return;
 		}
 
-		$this->resources = $this->getMockBuilder('WP_Rocket\Engine\Optimization\RUCSS\Database\Tables\Resources')
-			->disableOriginalConstructor()
-			->getMock();
-
 		$this->usedCSS = $this->getMockBuilder('WP_Rocket\Engine\Optimization\RUCSS\Database\Tables\UsedCSS')
```
Comment (CrochetFeve0251) [inline tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Database/dropRucssDatabaseTables.php]: Here too

File: tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Database/truncateUsedCssTable.php [hunk 4]
```diff
@@ -25,14 +23,10 @@ public function setUp() : void {
 			return;
 		}
 
-		$this->resources = $this->getMockBuilder('WP_Rocket\Engine\Optimization\RUCSS\Database\Tables\Resources')
-			->disableOriginalConstructor()
-			->getMock();
-
 		$this->usedCSS = $this->getMockBuilder('WP_Rocket\Engine\Optimization\RUCSS\Database\Tables\UsedCSS')
```
Comment (CrochetFeve0251) [inline tests/Unit/inc/Engine/Optimization/RUCSS/Admin/Database/truncateUsedCssTable.php]: Here too

**Review by CrochetFeve0251 — CHANGES_REQUESTED** [review]

**Review by CrochetFeve0251 — APPROVED** [review]

**Review by vmanthos — APPROVED** [review]
This is working as expected. 👍

**Pull request merged**